backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5710990_sys_AD_WF_Approval_Related_Translations.sql

```diff
@@ -0,0 +1,139 @@
+-- Element: ContentType
+-- 2023-11-20T08:57:02.728244600Z
+UPDATE AD_Element_Trl SET Name='File type', PrintName='File type',Updated=TO_TIMESTAMP('2023-11-20 10:57:02.728','YYYY-MM-DD HH24:MI:SS.US'),UpdatedBy=100 WHERE AD_Element_ID=543391 AND AD_Language='en_US'
+;
+
+-- 2023-11-20T08:57:02.761244600Z
+/* DDL */  select update_TRL_Tables_On_AD_Element_TRL_Update(543391,'en_US') 
+;
+
+-- Element: ContentType
+-- 2023-11-20T08:57:14.042027600Z
+UPDATE AD_Element_Trl SET Name='Dateityp', PrintName='Dateityp',Updated=TO_TIMESTAMP('2023-11-20 10:57:14.042','YYYY-MM-DD HH24:MI:SS.US'),UpdatedBy=100 WHERE AD_Element_ID=543391 AND AD_Language='de_CH'
+;
+
+-- 2023-11-20T08:57:14.044026Z
+/* DDL */  select update_TRL_Tables_On_AD_Element_TRL_Update(543391,'de_CH') 
+;
+
+-- Element: ContentType
+-- 2023-11-20T08:57:22.386084600Z
+UPDATE AD_Element_Trl SET Name='Dateityp', PrintName='Dateityp',Updated=TO_TIMESTAMP('2023-11-20 10:57:22.385','YYYY-MM-DD HH24:MI:SS.US'),UpdatedBy=100 WHERE AD_Element_ID=543391 AND AD_Language='de_DE'
+;
+
+-- 2023-11-20T08:57:22.387086200Z
+UPDATE AD_Element SET Name='Dateityp', PrintName='Dateityp' WHERE AD_Element_ID=543391
+;
+
+-- 2023-11-20T08:57:22.762586100Z
+/* DDL */  select update_ad_element_on_ad_element_trl_update(543391,'de_DE') 
+;
+
+-- 2023-11-20T08:57:22.764084600Z
+/* DDL */  select update_TRL_Tables_On_AD_Element_TRL_Update(543391,'de_DE') 
+;
+
+-- Process: AD_WF_Approval_Request_Approve(de.metas.workflow.process.AD_WF_Approval_Request_Approve)
```

Comment:
what's the purpose of this migration script?

mainly i am asking because i see things like AD_WF_Approval_Request_Approve which shall not be here